rollup.config.js

```diff
@@ -41,5 +41,6 @@ export default {
       ],
       exclude: "node_modules/**",
     }),
-  ]
+  ],
+  external: ["@hotwired/stimulus"],
```

Comment:
Why did you add this? Not needed with your changes, right?

Reply:
yeah you are right it shouldn' be needed.